builder/lxc/lxc_config.go

```diff
@@ -0,0 +1,43 @@
+package lxc
+
+import (
+	"io/ioutil"
+	"regexp"
+	"strings"
+)
+
+type LxcConfig struct {
+	filePath string
+	lines    []string
+}
+
+func NewLxcConfig(path string) (*LxcConfig, error) {
+	input, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	lines := strings.Split(string(input), "\n")
+	return &LxcConfig{path, lines}, nil
+}
+
+func (c *LxcConfig) SetRootFs(path string) {
+	c.SetProp("lxc.rootfs", path)
+}
+
+func (c *LxcConfig) SetProp(key string, value string) {
```

Comment:
I'm a go noob and didn't know it could distinguish public/private scope on a language level. TIL!